bundle/metadata/dependencies.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+  - type: olm.package
+    value:
+      packageName: redhat-openshift-pipelines
+      version: "1.5.2"
```

Comment:
This needs to be a range. '>=1.5.2' should work. Or, we can use an olm.gvk type of dependency if we don't necessarily want to specify the version.

Also, needs a newline at the end. :)

Reply:
If GVK type of dependency is available, I vote we go that route, since I don't really think the version matters to us. If for some reason there was a breaking change in a version of the pipelines-operator, I would assume that the team making the tekton tasks would take care of that, and we can be agnostic to version. Just my thoughts.

Reply:
I had the impression the that GVK is not fully supported. But since @bcrochet found the recipe for that I'll use that one.

Reply:
This one is already changed to @bcrochet suggestion. Let me know if it works now.